index.ts

```diff
@@ -112,11 +112,6 @@ function convertMarkdown(markdown: string) {
 
 const features: { [key: string]: FeatureData } = {};
 for (const [key, data] of yamlEntries('features')) {
-    // Draft features reserve an identifier but aren't complete yet. Skip them.
-    if (data.draft) {
-        continue;
-    }
-
```

Comment:
I think the crawler is recursive by default, so we still need a `draft` key or exclude the draft folder from the search.